src/simulator/base.py

```diff
@@ -0,0 +1,55 @@
+import object
+import position
+import shape
+import object_registry
+import numpy as np
+
+
+class Base(object.Object):
+    """
+    A physical station, primarily in charge of communicating with robots
+    and facilitating their navigation by transmitting their (estimated) distances with the base.
+    """
+
+    def __init__(self, oid: object.ObjectId, name: str, shape: shape.Shape, position: position.Position,
+                 owner_object: object.Object, registry: object_registry.ObjectRegistry,
+                 standard_deviation: float, time_using: float):
+        super().__init__(oid, name, shape, position, owner_object, registry)
+        self.standard_deviation = standard_deviation
```

Comment:
type hint these three variables

Reply:
except `self.__timer` ,which variable do you mean? I added type hint for other before.